mlir/test/Dialect/Vector/vector-rewrite-narrow-types.mlir

```diff
@@ -300,16 +298,25 @@ func.func @aligned_trunci_2d(%a: vector<8x32xi32>) -> vector<8x32xi4> {
 // CHECK-NOT:       vector.shli
 // CHECK-NOT:       vector.ori
 // CHECK:           arith.trunci
+// CHECK:           vector.deinterleave
   %0 = arith.trunci %a : vector<8x32xi32> to vector<8x32xi4>
   return %0 : vector<8x32xi4>
 }
 
+// CHECK-LABEL: func.func @aligned_trunci_nd(
+func.func @aligned_trunci_nd(%a: vector<3x8x32xi32>) -> vector<3x8x32xi4> {
```

Comment:
Why add a test for ND vectors for `trunci` but not for other operations?

Reply:
Since the ```trunci``` tests were mostly affected by the deinterleave change. An addition test was added to complete the coverage of dimensions tested (i.e. 1D, 2D, ND).